gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -178,6 +178,16 @@ public void onUpdateSpec(Spec updatedSpec) {
   }
 
   public void orchestrate(Spec spec) throws Exception {
+    // Add below waiting because TopologyCatalog and FlowCatalog service can be launched at the same time
```

Comment:
Add a note here to remove this wait when sequencing of service startup is introduced.